sentry/src/main/java/io/sentry/connection/AsyncConnection.java

```diff
@@ -172,7 +172,7 @@ public void run() {
                 // The current thread is managed by sentry
                 actualConnection.send(event);
             } catch (LockedDownException e) {
-                lockdownLogger.warn("The connection to Sentry is currently locked down.", e);
+                lockdownLogger.warn(e.getMessage());
```

Comment:
I would always suggest logging the full exception at DEBUG level for troubleshooting.

Reply:
Good call.